src/scripts/main.js

```diff
@@ -1,7 +1,129 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+function keyListener(e) {
+  if (game.getStatus() !== 'playing') {
+    return;
+  }
 
-// Write your code here
+  switch (e.key) {
+    case 'ArrowDown':
+      game.moveDown();
+      break;
+    case 'ArrowLeft':
+      game.moveLeft();
+      break;
+    case 'ArrowRight':
+      game.moveRight();
+      break;
+    case 'ArrowUp':
+      game.moveUp();
+      break;
+  }
+
+  update();
```

Comment:
Consider debouncing the `update` function to avoid performance issues when multiple keys are pressed in rapid succession.